### ISSUE TITLE
Tooltip not working for icons (e.g. warning icons)

### ISSUE THREAD

**Issue by micmro** [micmro/PerfCascade#210]
The icons used to show title tooltips, but since the keyboard access (same as #203) this does not work anymore.

<img width="112" alt="screen shot 2017-06-30 at 2 31 27 pm" src="https://user-images.githubusercontent.com/1680390/27722172-d21f5bbc-5da0-11e7-8324-582102def2ec.png">
